### PR TITLE
Added Main Carousel

### PR DESCRIPTION
**Pull request by capsule11**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,7 +13,9 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-icons": "^5.0.1",
-    "react-router-dom": "^6.21.3"
```
Comment (Sreejit-Sengupto) [inline package.json:16]: Ye hatae the kya?

**Comment by Sreejit-Sengupto** [comment]
Theek to lag raha🤔

File: src/components/Home/Home.jsx [hunk 2]
```diff
@@ -1,7 +1,13 @@
 import React from "react";
+import MainCarousel from "./MainCarousel";
 
 const Home = () => {
-  return <div>Home page goes here</div>;
+  return (
+    <>
+      <div></div>
```
Comment (Sreejit-Sengupto) [inline src/components/Home/Home.jsx:7]: Ye div khali kyu hai?

**Review by Sreejit-Sengupto — CHANGES_REQUESTED** [review]
Ye useEffect ka kam hai kya ab? React Slick me to autoplay hai. To isko hara do

**Comment by atulya-srivastava** [comment]
nice

**Comment by atulya-srivastava** [comment]
> Theek to lag raha🤔 ghar jake merge karke dekhte hai apne me

acha aisa kya

**Comment by Sreejit-Sengupto** [comment]
> > Theek to lag raha🤔 ghar jake merge karke dekhte hai apne me
> 
> acha aisa kya

**Pull request merged**